network/endpoint_windows.go

```diff
@@ -399,6 +417,9 @@ func (nw *network) newEndpointImplHnsV2(cli apipaClient, epInfo *EndpointInfo) (
 		ContainerID:              epInfo.ContainerID,
 		PODName:                  epInfo.PODName,
 		PODNameSpace:             epInfo.PODNameSpace,
+		// SecondaryInterfaces:      make(map[string]*InterfaceInfo),
```

Comment:
we probably need this if we want to delete the hns network on stateful CNI, if we already convert the code to stateless, then let's remove this SecondaryInteraces field

Reply:
okay I'm removing this then since secondary interfaces is for the stateful case I believe